src/components/Chart/Chart.js

```diff
@@ -1,12 +1,29 @@
 import React, { memo } from 'react'
+import { useSelector } from 'react-redux'
+import { reduxSelectors } from '@ufx-ui/bfx-containers'
 import PropTypes from 'prop-types'
 
 import { CHART_URL, env } from '../../redux/config'
+
 import './style.css'
 
-const Chart = ({ market: { wsID, base, quote } }) => {
+const { getCurrencySymbolMemo } = reduxSelectors
+
+const Chart = ({
+  market: {
+    wsID, base, quote, isPerp, uiID: _uiID,
+  },
+}) => {
+  const getCurrencySymbol = useSelector(getCurrencySymbolMemo)
+
+  const uiID = isPerp ? _uiID : `${getCurrencySymbol(base)}/${getCurrencySymbol(quote)}`
```

Comment:
replace `${getCurrencySymbol(base)}/${getCurrencySymbol(quote)}` with 
`getPairFromMarket(market, getCurrencySymbol)` so its easily changed and maintained from one place when we want to update labelling in future